environment/src/main/java/com/sequenceiq/environment/environment/validation/network/gcp/GcpEnvironmentNetworkValidator.java

```diff
@@ -22,19 +27,28 @@
 import com.sequenceiq.environment.network.CloudNetworkService;
 import com.sequenceiq.environment.network.dto.GcpParams;
 import com.sequenceiq.environment.network.dto.NetworkDto;
+import com.sequenceiq.environment.platformresource.PlatformParameterService;
+import com.sequenceiq.environment.platformresource.PlatformResourceRequest;
 
 @Component
 public class GcpEnvironmentNetworkValidator implements EnvironmentNetworkValidator {
 
     static final String MULTIPLE_AVAILABILITY_ZONES_PROVIDED_ERROR_MSG = "The multiple availability zones feature isn't available for GCP yet. "
             + "Please configure only one zone on field 'availabilityZones'";
 
+    static final String INVALID_ZONE_PATTERN = "The requested region '%s' doesn't contain the requested '%s' availability zone(s), available zones: '%s'";
+
+    static final String INVALID_REGION_PATTERN = "The environment's requested region '%s' doesn't exist on GCP side.";
```

Comment:
if you use the same pattern in the unit test, the test wouldn't break if somebody accidentally mess this text up

Reply:
Ok, I can be more strict and duplicate it in the UT to avoid such an accidental situation :slightly_smiling_face:

Reply:
thanks